style.css

```diff
@@ -1,19 +1,14 @@
 body {
```

Comment:
- [ ] Please check the sizes that you are applying to your app, as right now you are having a lot of horizontal scrolls that makes the app not look good and may lead to a bad user experience.

Here's the image for reference:
![Captura de pantalla 2022-02-20 013349](https://user-images.githubusercontent.com/59100477/154823865-8cd51353-fe2d-4ab2-b1f5-82deb7d3f44e.jpg)

_Note: I commented on the `body` tag because this is the main container of your app, it doesn't mean that the problem is in the `body`, so please make sure to find where the problem is and fix this issue._